package.json

```diff
@@ -15,6 +15,7 @@
     "babel-preset-env": "^1.7.0",
     "babel-register": "^6.26.0",
     "body-parser": "^1.18.3",
+    "celebrate": "^10.0.1",
```

Comment:
actually, I do not think adding this dependency is justified:

1. it is used only to check that an array contain a string and for testing a regex
2. the middleware render ugly json error which is not usefull:

![image](https://user-images.githubusercontent.com/4190728/68236279-2a231480-0005-11ea-8b6c-d1e3ee4fe377.png)

Reply:
Yes, long thinking about it : 
This update is a side project. the main ticket is the legacy France Connect. this update was there to have a full cinematic of the logout process. So...
we have to do it clean and fast. DRY and sugar. We know that this will be the first step of other error management and to avoid 'reinvent the wheel', we have to use a easy to implement solution (... and easy to remove if require). Create our own middleware is a loss of time and energy to get a small result. 
Concerning the test itself, the most important thing is the use of [Joi](https://github.com/hapijs/joi). the most famous input validator library on nodejs (easy to implement, easy to remove, great community).
Concerning the error 400, it is a very rare case and this is a automatic response from Joi validator formated for the web. We can override it if it is necessary but nothing was defined about it, another time another ticket !

Reply:
remove celebrate and move validate Joi to controller.